jetty-http2/http2-server/src/main/config/etc/jetty-http2.xml

```diff
@@ -9,6 +9,17 @@
         <Set name="maxConcurrentStreams"><Property name="jetty.http2.maxConcurrentStreams" deprecated="http2.maxConcurrentStreams" default="128"/></Set>
         <Set name="initialStreamRecvWindow"><Property name="jetty.http2.initialStreamRecvWindow" default="524288"/></Set>
         <Set name="initialSessionRecvWindow"><Property name="jetty.http2.initialSessionRecvWindow" default="1048576"/></Set>
+        <Set name="maxSettingsKeys"><Property name="jetty.http2.maxSettingsKeys" default="64"/></Set>
+        <Set name="rateControl">
+          <New class="org.eclipse.jetty.http2.parser.WindowRateControl">
+            <Arg type="int"><Property name="jetty.http2.rateControl" default="20"/></Arg>
```

Comment:
No, `20` is not "rateControl", it's the "rateControlWindow", so modify the property to be `jetty.http2.rateControlWindow`.

Consider modifying `WindowRateControl` to have a static factory method `ofSecond(long)` so that the XML will be shorter (no need to create the `Duration` object).